### PR TITLE
feat(application-system-api): Application system api/improve devx

### PR DESCRIPTION
**Pull request by kristofer-lp8f**
## What

Improve development experience (devx) by bundling together required steps to get the application-system-form and api up and running.

## Why

Getting the application system up and running is a pain for every new developer and after fresh installs. There are likely more modules/apps which could be improved, but this is where I have experience.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: workspace.json [hunk 1]
```diff
@@ -990,6 +990,15 @@
             "passWithNoTests": true
           },
           "outputs": ["coverage/libs/api/domains/driving-license"]
+        },
+        "dev": {
+          "builder": "@nrwl/workspace:run-commands",
+          "options": {
+            "commands": [
+              "kubectl -n socat port-forward svc/socat-xroad 8081:80"
```
Comment (eirikurn) [inline workspace.json:998]: This should use our `scripts/run-xroad-proxy.sh` script.

File: workspace.json [hunk 2]
```diff
@@ -1741,7 +1750,33 @@
...
+        },
+        "dev": {
+          "builder": "@nrwl/workspace:run-commands",
+          "options": {
+            "commands": [
+              "yarn start api",
+              "yarn start application-system-api",
+              "yarn start application-system-form",
+              "kubectl port-forward svc/socat-soffia 8443:443 -n socat",
+              "kubectl port-forward svc/socat-xroad 8081:80 -n socat"
```
Comment (eirikurn) [inline workspace.json:1775]: This should use our `scripts/run-*-proxy.sh` scripts.

**Review by eirikurn — COMMENTED** [review]
Should these be documented somewhere?

I'm not a bit conflicted about these `dev` helpers. They'll quickly break if you need to run more than one `dev` commands if they try and start some of the same things. It's also easy to forget about these when we change stuff.

I empathise with the learning overhead and labour of running all these commands, but it's always good to learn about all the cogs of our systems. I would at least document all this stuff in README before creating (and explaining) helper scripts which runs everything.

File: workspace.json (hunk 2 above)
Comment (kristofer-lp8f) [reply workspace.json:1775]: Which proxy script does `svc/socat-soffia` correspond to?

File: apps/application-system/README.md [hunk 3]
```diff
@@ -10,6 +10,18 @@ This project forms the base for all business applications belonging to island.is
 - [Staging](https://beta.staging01.devland.is/umsoknir/)
 - [Production](https://island.is/umsoknir/)
 
+
+## Quickstart
```
Comment (rafnarnason) [inline apps/application-system/README.md:14]: following up on similar notes like @eirikurn I would suggest that the documentation in the quickstart section would indicate in more detail that they are a bundle of commands which are also outlined in the readme

File: workspace.json (hunk 2 above)
Comment (kristofer-lp8f) [reply workspace.json]: Added the script here 🙄

File: apps/application-system/README.md (hunk 3 above)
Comment (kristofer-lp8f) [reply apps/application-system/README.md]: Updated. How do you like it now?

**Review by pshomov — APPROVED** [review]
LGTM

**Comment by kristofer-lp8f** [comment]
@island-is/core Can someone review this? Is a new review needed after every main update?

**Review by albinagu — APPROVED** [review]

**Comment by kristofer-lp8f** [comment]
@eirikurn I updated the README. Is this good enough until these scripts become obsolete? I set them up in parallel, so one failing does not fail the rest, at least for `:dev`

**Review by rafnarnason — APPROVED** [review]
lgtm 🍕

**Review by eirikurn — APPROVED** [review]
LGTM 👍

**Pull request merged**